appcenter-analytics/android/build.gradle

```diff
@@ -21,6 +21,6 @@ dependencies {
     compile 'com.facebook.react:react-native:+'
     compile 'com.microsoft.appcenter:appcenter-analytics:1.10.0'
 
-    //compile project(':AppCenterReactNativeShared')   // For testing with TestApp
-    compile 'com.microsoft.appcenter.reactnative:appcenter-react-native:1.10.0'
+    compile project(':AppCenterReactNativeShared')   // For testing with TestApp
```

Comment:
revert this change.

Reply:
Won't be able to compile if I do that, we need to keep that until release.

Reply:
Your feedback is addressed in https://github.com/Microsoft/AppCenter-SDK-React-Native/pull/480/commits/a3adae58e3a62b77aa2185e7ff974beb9a24c3db